docs/building/npcs.md

```diff
@@ -22,6 +22,9 @@ In Ranvier all NPCs for an area are defined in a single file within the area fol
   keywords: ["wise", "old", "man"]
   name: "Wise Old Man"
   description: "A wise looking old man sits on the ground with legs crossed."
+  equipment:
+    head: "a duffy old hat"
```

Comment:
The value here should be an entityReference, not an arbitrary string

Reply:
updated!